### PR TITLE
change items.py to add float and int

### PR DESCRIPTION
**Pull request by SamirPS**
I change the add method for Integrer and add a test from this issue #214  now it's work but i don't know if you need this

### CODE REVIEW

**Comment by SamirPS** [comment]
@frostming @abravalheri @sdispater @hukkinj1 @Secrus

File: tomlkit/items.py [hunk 1]
```diff
@@ -664,7 +663,7 @@ def __rsub__(self, other):
 
     def _new(self, result):
         raw = str(result)
-
+
```
Comment (frostming) [inline tomlkit/items.py:666]: Redundant indentation.

Comment (SamirPS) [reply tomlkit/items.py:666]: What line do i need to remove @frostming , i don't see the problem

Comment (frostming) [reply tomlkit/items.py:666]: Line 666, whitespaces should be trimmed

Comment (SamirPS) [reply tomlkit/items.py]: good now ?

Comment (SamirPS) [reply tomlkit/items.py]: @frostming

**Comment by SamirPS** [comment]
@frostming

File: tomlkit/items.py [hunk 2]
```diff
@@ -637,9 +637,8 @@ def as_string(self) -> str:
         return self._raw
 
     def __add__(self, other):
-        result = super().__add__(other)
 
-        return self._new(result)
+        return self._new(int(self._raw)+other)
```
Comment (frostming) [inline tomlkit/items.py:641]:
```suggestion
        return self._new(int(self._raw) + other)
```

**Comment by SamirPS** [comment]
I just add your commit @frostming

**Comment by frostming** [comment]
> I add your commit @frostming 

Thanks, as a kind note about open source contribution. Don't ping someone with @ but only click the resolve button when you are done with the change. It is a useless noise and will make people feel disturbed.

**Comment by SamirPS** [comment]
Sorry i didn't know

File: tests/test_items.py [hunk 3]
```diff
@@ -561,6 +561,12 @@ def test_item_array_of_dicts_converted_to_aot():
 """
     )
```
Comment (frostming) [inline tests/test_items.py:563]:
```suggestion


```

**Comment by frostming** [comment]
Thanks for your contribution.

**Pull request merged**